docs/fundamentals/executor/hub/query-executor.md

```diff
@@ -0,0 +1,12 @@
+(query-executor)=
+# Query
```

Comment:
This should not be its own session but rather be part of the Publish executor part

Reply:
Exactly, should be part of `push-executor`.